Penteract/Assets/Scripts/GodModeController.cpp

```diff
@@ -112,10 +114,26 @@ void GodModeController::OnChildToggle(unsigned int index, bool isChecked) {
 		break;
 	case 5:
 		if (isChecked) {
-			if (invincible) invincible->Enable();
+			if (playerControllerScript) playerControllerScript->SetOverpower(true);
```

Comment:
You could just if (ischecked) before the switch so you don't need to repeat it in every case